grobid-core/src/main/java/org/grobid/core/jni/PythonEnvironmentConfig.java

```diff
@@ -44,6 +44,23 @@ public Path getSitePackagesPath() {
         return this.sitePackagesPath;
     }
 
+    public Path getNativeLibPath() {
+        if (this.virtualEnv == null) {
+            return null;
+        }
+        return Paths.get(this.virtualEnv.toString(), "lib");
```

Comment:
This is really only useful for conda virtual envs. But it doesn't really matter if we add a directory without libraries.